tests/conftest.py

```diff
@@ -92,8 +94,93 @@ def env():
         shutil.rmtree(fake_home, ignore_errors=True)
 
 
+@pytest.fixture()
+def webapp(active_config):
+    from dallinger.experiment_server import sockets
+    app = sockets.app
+    app.config['DEBUG'] = True
+    app.config['TESTING'] = True
+    client = app.test_client()
+    yield client
+
+
+@pytest.fixture
+def a(db_session):
```

Comment:
Could this be called something more descriptive, like `factory`?

Reply:
It's named this way because (I think) it reads well when _used_, and it's nice to have it very short since it gets inlined a lot:
```
node = a.node(network=a.star_network(), participant=a.participant())
```
I'd argue that the shortness of the name is (maybe?) further excused by the fact that, as a global fixture that's the standard way to instantiate model objects in tests, it becomes familiar and well-understood. Persuaded?

(Full disclosure: this isn't my pattern; I stole it from my current favorite book on testing "Working Effectively with Unit Tests" by Jay Fields)